--- src/gatk4/gatk	2023-05-01 15:58:54.493104181 +0200
+++ dest/gatk4/gatk	2023-05-01 15:58:33.275056659 +0200
@@ -27,6 +27,9 @@
 
 projectName = "gatk"
 
+#MV 1-05-2023 alternate JRE location/
+JAVA_BIN = "/lib/jvm/jre-17/bin/java"
+
 BUILD_LOCATION = script +"/build/install/" + projectName + "/bin/"
 GATK_RUN_SCRIPT = BUILD_LOCATION + projectName
 GATK_LOCAL_JAR_ENV_VARIABLE = "GATK_LOCAL_JAR"
@@ -237,7 +240,8 @@
 
 
 def formatLocalJarCommand(localJar):
-    return ["java"] + PACKAGED_LOCAL_JAR_OPTIONS + [ "-jar", localJar]
+    return [ JAVA_BIN ] + PACKAGED_LOCAL_JAR_OPTIONS + [ "-jar", localJar]
+#    return ["java"] + PACKAGED_LOCAL_JAR_OPTIONS + [ "-jar", localJar]
 
 def getGatkWrapperScript(throwIfNotFound=True):
     if not os.path.exists(GATK_RUN_SCRIPT):
